PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -136,6 +138,19 @@ public string GetCustomUI(string ribbonId)
         #region Ribbon Callbacks
         //Create callback methods here. For more information about adding callback methods, select the Ribbon XML item in Solution Explorer and then press F1
 
+        // Set the visibility of the Drawing Tools Format Tab
+        public bool SetVisibleFormatTab(Office.IRibbonControl control)

```

Comment:
This method name is misleading since it doesnt set

Reply:
Actually it does set the visibility of the format tab, just that the setting is not up to a direct input from the user, but dependent on the status of the checkbox

Reply:
Would it be better to call it toggle instead of set? Since without a parameter set may sound like setting it to true everytime